base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -450,46 +450,120 @@ where
     pub async fn query_unspent_outputs_status(
```

Comment:
Few methods are public when they don't need to be?

Reply:
you are right, in fact almost all these functions don't need to be pub. A hold over from before the service model. Will remove.